include/re_sa.h

```diff
@@ -59,5 +59,12 @@ bool     sa_is_linklocal(const struct sa *sa);
 bool     sa_is_loopback(const struct sa *sa);
 bool     sa_is_any(const struct sa *sa);
 
+#ifdef HAVE_INET6
+void     sa_set_scopeid(struct sa *sa, int scopeid);
```

Comment:
the scope_id is of type `uint32_t` in the posix api...

should we use the same ?

Reply:
Ah, yes!